website/docs/docs/cloud/secure/vcs-privatelink.md

```diff
@@ -0,0 +1,79 @@
+- **Endpoint Service** - The VPC Endpoint Service is what allows for the VPC to VPC connection, routing incoming requests to the configured load balancer.
+    - **Load balancer type:** Network.
+    - **Load balancer:** Attach the NLB created in the previous step.
+    - **Acceptance required (recommended)**:  When enabled, requires a new connection request to the VPC Endpoint Service to be accepted by the customer before connectivity is allowed ([details](https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#accept-reject-connection-requests)).
+
+    Once these resources have been provisioned, access needs to be granted for the dbt Labs AWS account to create a VPC Endpoint in our VPC. On the newly created VPC Endpoint Service, add a new [Allowed Principal](https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#add-remove-permissions) for the appropriate dbt Labs principal: 
+
+   -  **AWS Account ID:**	`arn:aws:iam::<account id>:root` (contact your dbt Labs account representative for appropriate account ID).
+
+### Completing the connection
+
+To complete the connection, dbt Labs must now provision a VPC Endpoint to connect to your VPC Endpoint Service. This requires you send the following information:
+ 
+ - VPC Endpoint Service name:
+
+ <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/vpc-endpoint-service-name.png" width="80%" title="Location of the VPC Endpoint Service name in the AWS console" />
+
+ - **DNS configuration:** If the connection to the VCS service requires a custom domain and/or URL for TLS, a private hosted zone can be configured by the dbt Labs Infrastructure team in the dbt Cloud private network. For example:
+    - **Private hosted zone:** `examplecorp.com`
+    - **DNS record:** `github.examplecorp.com`
+
+### Accepting the connection request
+
+When you have been notified that the resources are provisioned within the dbt Cloud environment, you must accept the endpoint connection (unless the VPC Endpoint Service is set to auto-accept connection requests). Requests can be accepted through the AWS console, as seen below, or through the AWS CLI.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/accept-request.png" width="80%" title="Accept the connection request" />
+
+Once the endpoint connection request is accepted, the PrivateLink endpoint is ready to use in dbt Cloud.
+
+## Configure in dbt Cloud
+
+Once the dbt Labs team confirms that the PrivateLink integration is complete, you can select it as part of a new or existing git configuration. Select **PrivateLink Endpoint** as the connection type, and your configured integrations will appear in the dropdown menu. This is a manual process; the existing connections remain configured as they were until this step is complete.  
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/vcs-setup-new.png" width="80%" title="Configuring a new git integration" />
```

Comment:
It's unclear to me what I should pay attention to in these screens as they kind of bleed together. Perhap they should be side by side and have a more descriptive title?

Reply:
The whole screenshot is important - is there something, in particular, causing confusion? They're selecting the proper resources for a new and existing integration

Reply:
@matthewshaver I think the lack of clear steps confused me so maybe it's better now. I also wish the screen shot title preceded the screen shots because I believe that would also help you glance over them to see the purpose of each. The "Configuring a new git integration" title kind of gets lost in the middle and it wasn't clear whether it was a title within the screen shot.